src/resources/js/routes/Administration.svelte

```diff
@@ -0,0 +1,328 @@
+
+      // Fetch all meetings at once
+      const existingMeetings = await RootServerApi.getMeetings({
+        meetingIds: meetingIds.map((i) => i.toString()).join(',')
+      });
+
+      console.log(`Retrieved ${existingMeetings.length} existing meetings`);
+
+      // Create a map of existing meetings for quick lookup
+      const existingMeetingsMap = new Map();
+      existingMeetings.forEach((meeting) => {
+        existingMeetingsMap.set(meeting.id, meeting);
+      });
+
+      // Process each meeting
+      for (const meetingId of meetingIds) {
+        const newWorldId = updateMap.get(meetingId);
+        const existingMeeting = existingMeetingsMap.get(meetingId);
+
+        if (!existingMeeting) {
+          if (newWorldId?.toLowerCase() === 'deleted') {
+            stats.notedAsDeleted.push(meetingId);
+          } else {
+            console.warn(`Meeting ${meetingId} not found`);
+            stats.notFound.push(meetingId);
+          }
+          continue;
+        }
+
+        // Check if update is needed
+        if (existingMeeting.worldId === newWorldId || (existingMeeting.worldId === null && newWorldId === '')) {
+          console.log(`Meeting ${meetingId} already has committee ${newWorldId} - no update needed`);
+          stats.noUpdateNeeded.push(meetingId);
+          continue;
+        }
+
+        try {
+          const updatedValues: MeetingPartialUpdate = {
+            worldId: newWorldId
+          };
+          await RootServerApi.partialUpdateMeeting(meetingId, updatedValues);
+          console.log(`Successfully updated meeting ${meetingId}: ${existingMeeting.worldId} → ${newWorldId}`);
+          stats.updated.push(meetingId);
+        } catch (err) {
+          console.error(`Failed to update meeting ${meetingId}:`, err);
+          stats.errors.push(meetingId.toString() + ' ' + err);
+          errorMessage = `Failed to update meeting ${meetingId}: ${err}`;
+        }
+      }
+
+      console.log('Processing complete:', $state.snapshot(stats));
+    } catch (err) {
+      errorMessage = err instanceof Error ? err.message : 'An error occurred while processing the file';
+      console.error('Error processing file:', err);
+    } finally {
+      isProcessed = true;
+      isLoading = false;
+    }
+  }
+
+  function handleProcessFile() {
+    if (files && files.length > 0) {
+      processFile(files[0]);
+    }
+  }
+
+  function getFileExtension(filename: string): string {
+    return filename.toLowerCase().split('.').pop() || '';
+  }
+
+  function isValidFileType(extension: string): boolean {
+    return ['csv', 'xlsx'].includes(extension);
+  }
+
+  function meetingIdsToString(ids: number[]): string {
+    return ids
+      .slice()
+      .sort((a, b) => a - b)
+      .map((m) => m.toString())
+      .join(', ');
+  }
+
+  // Reset processed state when new files are selected
+  $effect(() => {
+    if (files && files.length > 0) {
+      isProcessed = false;
+      stats = {
```

Comment:
I think we also want to add `errorMessage = null;` here too, to reset errors when new file is selected